src/Symfony/Component/Form/Tests/Extension/Validator/Constraints/FormValidatorTest.php

```diff
@@ -149,11 +127,11 @@ public function testValidateIfChildWithValidConstraint()
         $this->assertNoViolation();
     }
 
-    public function testDontValidateIfParentWithoutCascadeValidation()
+    public function testDontValidateIfParentWithoutValidConstraint()
```

Comment:
I am not sure, I thought it might be intended as it looks like the test for an explizit cascade without the option/constraint.